docs/logical-data-model.md

```diff
@@ -0,0 +1,80 @@
+# Logical data model
+
+![Resource](./assets/logical-data-model-1.png)
+
+## Resource
+
+| Key | Value description |
+| --- | ----------------- |
+| `id` | A unique resource id. |
+| `parentId` | The parent resource id. |
+| `version` | Resource version. |
+| `title` | The name (title) of the resource. Rendered in the UI. |
+| `description` | The description of the resource. Rendered in the UI. |
+| `resourceType` | Resource type. Workspace or Service. |
+| `status` | Resource status. |
+| `markedForDeletion` | Is the resource marked for deletion. |
+| `properties` | Resource properties list.|
```

Comment:
Currently we only have name/value pairs sent in (as requested by the spec) - so in the current implementation this is a dict (name/value). But we can leave as is, this might be where we need to go eventually